ailab/benchmark/db_utils.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+##############################################################################
+# Copyright 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+##############################################################################
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+import ast
+import json
+import os
+import requests
+import sys
+from time import sleep
+
+# from .custom_logger import getLogger
+
+
+def getBenchmarks(bfile, framework=None):
```

Comment:
Is this method used here?

Reply:
Removed the entire file from this commit.